### ISSUE TITLE
Choice option in column filter: instead of distinc values from db table only displayed data

### ISSUE THREAD

**Issue by doctsh** [morlandi/django-ajax-datatable#90]
Hi
I have started using this wonderful package in my application. I am stuck with one place where choices are enabled and it shows models field choices or db table distinct values. 

But when data is filtered for a user if we can restrict the column dropdown options to distinct values of what is being served through initial queryset will be more opt than collecting from entire db.

Is there anyway to achieve this? or am I doing anything wrong?

**Comment by morlandi** [comment]
Hello,
you could try settings `autofilter = True` which does exactly what you want:

https://github.com/morlandi/django-ajax-datatable#id12

but it might be ignored when Field has `chioices` ... I'm not sure about this details.
I'ld give it a try

**Comment by doctsh** [comment]
Thank you so much for prompt reply.
If I remove choices then autofilter does not work (filter becomes textbox), if I include choices then autofilter takes unique db field....

**Comment by morlandi** [comment]
The idea behind this is that if you start to fill a brand new table, no values are available yet, thus the only chance to help the user is to show the list of choices associated with the Field.
In my project I tend to use choices with a limited number of values, so it's not a big problem; when the number of choices is significant, I rather opt to have a separate "lookup" table and a FK.

I do undestand that there might be cases where this solution is not the optimal choice.

**Comment by doctsh** [comment]
Yes, you are right. I moved a filter out of column filtering and filter using select from shortlisted items:
![image](https://user-images.githubusercontent.com/5936743/177960749-875b1f4f-7b49-46a9-aeac-7debebb4b426.png)
